src/modules/home/Home.js

```diff
@@ -842,25 +871,28 @@ const Home = (props) => {
         onPress={(name, position) => dialogClickHandler('notebookPanelMenuVisible', name, position)}
         onTouchOutside={() => toggleDialog('notebookPanelMenuVisible')}
       />
-      <Modal
-        isVisible={props.isImageModalVisible}
-        useNativeDriver={true}
-        style={{flex: 1}}
-      >
-        <View style={homeStyles.modal}>
-          <Button
-            type={'clear'}
-            titleProps={{color: 'white'}}
-            title='Hide modal'
-            onPress={() => toggleImageModal()}/>
-          <Image
-            source={props.selectedImage
-              ? {uri: useImages.getLocalImageSrc(props.selectedImage.id)}
-              : require('../../assets/images/noimage.jpg')}
-            style={{width: wp('90%'), height: hp('90%')}}
+      {props.isImageModalVisible && <SafeAreaView>
```

Comment:
put <SafeAreaView in parentheses and move to next line so opening and closing tags lined up